etherman/etherman.go

```diff
@@ -489,6 +489,7 @@ func (etherMan *Client) updateL1InfoTreeEvent(ctx context.Context, vLog types.Lo
 		*blocks = append(*blocks, *block)
 	}
 	gExitRoot.PreviousBlockHash = block.ParentHash
+	gExitRoot.MinTimestamp = block.ReceivedAt
```

Comment:
```suggestion
	gExitRoot.MinTimestamp = time.Unix(int64(fullBlock.Time()), 0)
```

Reply:
In fact, is `fullBlock.Time()`, the function `prepareBlock` assign `Time()` to field `ReceivedAt`